cli_tools/diagnostics/main_windows_test.go

```diff
@@ -44,6 +47,50 @@ func fileNonExist(e error) bool {
 	return false
 }
 
+func TestGetPlainEventLogs(t *testing.T) {
+	// Test setup: create temp test folder for test, clean it up afterwards
+	var err error
+	tmpFolder, err = ioutil.TempDir("", "getPlainEventLogsTest")
+	if err != nil {
+		t.Errorf("Error creating a temporary test folder:\n%v", err.Error())
+	}
+	defer os.RemoveAll(tmpFolder)
+
+	tests := []struct {
+		name         string
+		args         []winEvt
+		want         []string
+		expectErrStr string
+	}{
+		{name: "Nil events", args: nil, want: []string{}, expectErrStr: ""},
+		{name: "Empty events", args: []winEvt{}, want: []string{}, expectErrStr: ""},
+		{name: "Existing events logName", args: []winEvt{{"Application", false}}, want: []string{filepath.Join(tmpFolder, applicationTextLogFileName)}, expectErrStr: ""},
```

Comment:
I don't know Go style at all, but it seems like some line breaks might make this long line more readable.